manifests/init.pp

```diff
@@ -205,6 +205,8 @@
   Integer[1,65536]                                     $redis_server_port,
   Integer[1]                                           $redis_timeout,
   Stdlib::Absolutepath                                 $unbound_conf_d,
+  Boolean                                              $manage_roothints_file,
```

Comment:
can you please add puppet-strings like documentation for the two new parameters?